components/BottomSheet/src/MDCBottomSheetPresentationController.m

```diff
@@ -168,7 +168,12 @@ - (void)viewWillTransitionToSize:(CGSize)size
 }
 
 - (void)updatePreferredSheetHeight {
-  CGFloat preferredContentHeight = self.presentedViewController.preferredContentSize.height;
+  CGFloat preferredContentHeight;
+  if (self.preferredSheetHeight == 0.f) {
```

Comment:
Please consider checking if the height is less than or equal to zero. I don't think we should present a view with a negative `preferredSheetHeight`.

Reply:
Done.